docs/release-process.md

```diff
@@ -105,23 +105,51 @@ on GitHub (beyond the sources automatically provided there). Instead we add
 a Downloads section that notes the exact tags and digests that the images can
 be found at on quay.io.
 
-Use the following partial snippet as an example:
-```
-Images built for this release can be obtained from the quay.io image registry.
-
-### samba-server
-* By tag: quay.io/samba.org/samba-server:v0.3
-* By digest: quay.io/samba.org/samba-server@sha256:09c867343af39b237230f94a734eacc8313f2330c7d934994522ced46b740715
-### samba-ad-server
-* By tag: quay.io/samba.org/samba-ad-server:v0.3
-* By digest: quay.io/samba.org/samba-ad-server@sha256:a1d901f44be2af5a516b21e45dbd6ebd2f64500dfbce112886cdce09a5c3cbd5
+The downloads section can be generated using the following shell script:
+```bash
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
I'm curious, why do you recommend that?

Reply:
I can only think of the [portability](https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability) use case. But this brings in a level of indirection. Given that I don't think we have considered the portability aspect elsewhere in samba-container repository and this hasn't been a problem till now.

just my 2 cents.

Reply:
OK, I made the change even though I didn't think it was very necessary. I just figured it would be less work to make the change than to argue it ;-)